### DISCUSSION TITLE
Everything is too small and tiny on a new i3 install

### DISCUSSION THREAD

**Discussion by raspacee** [i3/i3#5993] [Q&A]
I run a Ubuntu 22.04 on gnome. I just installed i3 on it.
But everything is too small and tiny.
After searching around the web,
I also tried running
`xrandr --dpi 120` on terminal.

I tried putting `exec --no-startup-id xrandr --dpi 120` on the i3
config file, then tried both reloading the i3 and logging out and logging in. Nothing happened.

I am new to ubuntu and i3 so having a hard time figuring things out

Running xrandr on terminal gives this output 

    Screen 0: minimum 320 x 200, current 1920 x 1080, maximum 16384 x 16384 
    eDP-1 connected primary 1920x1080+0+0 (normal left inverted right x axis y axis) 309mm x 175mm 
       1920x1080     60.02*+  40.02   
       1680x1050     60.02   
       1400x1050     60.02   
       1600x900      60.02   
       1280x1024     60.02   
       1400x900      60.02   
       1280x960      60.02   
       1440x810      60.02   
       1368x768      60.02   
       1280x800      60.02   
       1280x720      60.02   
       1024x768      60.02   
       960x720       60.02   
       928x696       60.02   
       896x672       60.02   
       1024x576      60.02   
       960x600       60.02   
       960x540       60.02   
       800x600       60.02   
       840x525       60.02   
       864x486       60.02   
       700x525       60.02   
       800x450       60.02   
       640x512       60.02   
       700x450       60.02   
       640x480       60.02   
       720x405       60.02   
       684x384       60.02   
       640x360       60.02   
       512x384       60.02   
       512x288       60.02   
       480x270       60.02   
       400x300       60.02   
       432x243       60.02   
       320x240       60.02   
       360x202       60.02   
       320x180       60.02   
    HDMI-1 disconnected (normal left inverted right x axis y axis)

**Comment by orestisfl** [comment]
Have you tried tips from section [8.3. High-resolution displays (aka HIDPI displays)](https://i3wm.org/docs/userguide.html#hidpi) from the user guide?
